db/models.py

```diff
@@ -14,3 +16,37 @@
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="actors")
+    genres = models.ManyToManyField(Genre, related_name="genres")
+
+    def __str__(self) -> CharField:
+        return self.title
+
+
+class CinemaHall(models.Model):
+    name = models.CharField(max_length=255)
+    rows = models.IntegerField()
+    seats_in_row = models.IntegerField()
+
+    def __str__(self) -> CharField:
+        return self.name
+
+    @property
+    def capacity(self) -> int:
+        return self.seats_in_row * self.rows
+
+
+class MovieSession(models.Model):
+    show_time = models.DateTimeField()
+    cinema_hall = models.ForeignKey(CinemaHall, related_name="sessions",
+                                    on_delete=models.CASCADE)
+    movie = models.ForeignKey(Movie, related_name="sessions",
```

Comment:
The `related_name` for the `movie` ForeignKey in the `MovieSession` model is set to 'sessions'. Ensure that this name does not conflict with other related names in your models. If it does, consider using a more specific name like 'movie_sessions'.